### DISCUSSION TITLE
AuthException(message: AuthRetryableFetchError, statusCode: null)

### DISCUSSION THREAD

**Discussion by divo662** [supabase/supabase#27597] [Feature Requests]
I started with supabase auth and I wasn't having any issues at first then suddenly I started getting that error which has unabled me to created new users but i can log in my users but can not create a user. i have tried different network provider, VPN and many other stuffs. this is my function for creating a user.

  Future<void> signUp() async {
    if (emailController.text.isEmpty || passwordController.text.isEmpty) {
      _showSnackBar("Please enter your email and password");
      return;
    }

    if (passwordController.text != confirmPasswordController.text) {
      _showSnackBar("Password doesn't match");
      return;
    }

    if (!_validatePassword(passwordController.text)) {
      _showSnackBar("Password must be more than 6 characters");
      return;
    }

    setState(() {
      _isLoading = true;
    });

    try {
      final response = await supabase.auth.signUp(
        email: emailController.text.trim(),
        password: passwordController.text.trim(),
        data: {
          'first_name': firstNameController.text.trim(),
          'last_name': lastNameController.text.trim(),
          'phone_number': phoneNumberController.text.trim(),
        },
      );

      if (response.session != null) {
        throw response.session!;
      }

      if (!mounted) return;

      NavigationHelper.navigateToPage(
        context,
        VerifySuccessScreen(navScreen: () {
          NavigationHelper.navigateToReplacement(context, const HomeScreen());
        }),
      );
    } on AuthException catch (e) {
      _handleAuthError(e);
    } on SocketException catch (_) {
      _showSnackBar("No internet connection. Please check your network.");
    } catch (e) {
      _showSnackBar("An unexpected error occurred. Please try again later.");
    } finally {
      setState(() {
        _isLoading = false;
      });
    }
  }

**Comment by ImKikiLin** [comment]
I'm having a similar problem. Did you ever get this working?
